pypeit/images/pypeitimage.py

```diff
@@ -78,12 +78,13 @@ def from_file(cls, file):
         return pypeitImage
 
     def __init__(self, image, ivar=None, rn2img=None, bpm=None,
-                 binning=None, crmask=None, mask=None):
+                 binning=None, crmask=None, mask=None, rawheadlst=None):
```

Comment:
Add a doc describing `rawheadlst`

Reply:
Done - I've also added docs for the other variables, I wasn't quite sure what `rn2img` is, but I guess it's a variance image of the read noise. Correct me if I'm wrong.